config/feature_flags.go

```diff
@@ -73,7 +74,8 @@ func (f *FeatureFlagSynchronizer) Close() {
 // featureFlagsFromMap sets the feature flags from a map[string]string.
 // It starts with baseFeatureFlags and only sets values that are
 // given by the upstream management system.
-// Makes the assumption that all feature flags are strings for now.
+// Makes the assumption that all feature flags are strings or booleans.
+// Strings are converted to booleans by consitering "on" or "true" as true and any other value as false.
```

Comment:
Wondering if we shouldn't use the `strconv.ParseBool()` semantics, since that's the Go standard way of parsing (and pretty well understood)? The downside of that is that `on` would return an error.

Reply:
The defaults for split.io are "on" and "off" so I would really like "on" to be true. What I can do is check for "on" then use strconv.